test/common_node/job_finished_test.cpp

```diff
@@ -0,0 +1,160 @@
+#include <gtest/gtest.h>
+
+#include <chrono>
+#include <cinttypes>
+#include <nlohmann/json.hpp>
+
+#include "rclcpp/logging.hpp"
+#include "rclcpp/node.hpp"
+#include "rclcpp/executors.hpp"
+
+#include "common_package/common_node.hpp"
+
+#include "interfaces/msg/job_finished.hpp"
+
+using namespace common_lib;
+
+class JobFinishedCommonNode : public CommonNode
+{
+public:
+    JobFinishedCommonNode(const std::string &name) : CommonNode(name)
+    {
+    }
+
+    inline void activate()
+    {
+        CommonNode::activate();
+    }
+
+    inline void job_finished(const uint8_t error_code, const nlohmann::json payload)
```

Comment:
Hier fehlt die Referenz noch.
```suggestion
    inline void job_finished(const uint8_t error_code, const nlohmann::json &payload)
```

Reply:
Ist erledigt.